tensorflow_datasets/core/features/class_label_feature.py

```diff
@@ -50,6 +53,7 @@ def __init__(self, num_classes=None, names=None, names_file=None):
     self._num_classes = None
     self._str2int = None
     self._int2str = None
+    self._is_multilabel = False
```

Comment:
If the user load labels from filename, you should set this to True. By checking the extension for instance.

```
features={
    'label': ClassLabel(names_file='label.csv'),
}
```
otherwise you will try to load the `.txt` file in `_load_names_from_file`